pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -607,7 +607,7 @@
         "4.4.0",
         "4.5.0"
       ],
-      "BaselineVersion": "4.4.0",
+      "BaselineVersion": "4.5.1",
```

Comment:
Should `BaselineVersion` be one of the `StableVersions`?

Reply:
No. What we want here is to force all packages that depend on this package, to get that dependency to use the new version that we are shipping, which for this one is 4.5.1. To illustrate, imagine we shipped both packages A and B, both versioned 1.0.0 and package B had a dependency to package A. Then, we came in and did this work for fixing packages that needed to cross compile, and A was one of the packages we fixed and ship as 1.0.1. If we don't make this change on the baseline version, when building package B it will still depend on version 1.0.0 of A, which is the wrong one. By changing baseline version to the version we will ship, then B will now depend on A version 1.0.1, and therefore B will also not need facades any longer when running on .NET Framework. I hope that sort of makes sense.